WORKSPACE

```diff
@@ -89,7 +90,7 @@ load("@build_bazel_rules_typescript//:defs.bzl", "ts_setup_workspace")
 ts_setup_workspace()
 
 # Setup the Sass rule repositories.
-load("@io_bazel_rules_sass//:defs.bzl", "sass_repositories")
+load("@io_bazel_rules_sass//sass:sass_repositories.bzl", "sass_repositories")
```

Comment:
AFAIK this should stay `:defs`. @DevVersion?

Reply:
Yeah, that's the new public API.

Reply:
You guys are right. It's the downstream repo that is out of date. Thanks.